challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,9 @@
+const BinarySearch = (arr, key) => {
```

Comment:
While this meet the requirements for the challenge this isn't technically a binary search. A binary search should cut the sorted array in half and check each half to see which one would include what we're searching for (i.e. if we're looking for a word check to see the alphabetical order). This solution iterates through the whole array.